WalletWasabi.Fluent/ApplicationStateManager.cs

```diff
@@ -227,4 +227,10 @@ void IMainWindowService.Shutdown(bool restart)
 		_restartRequest = restart;
 		_stateMachine.Fire(ApplicationViewModel.CanShutdown(_restartRequest) ? Trigger.ShutdownRequested : Trigger.ShutdownPrevented);
 	}
+
+	void IMainWindowService.BringToFront()
```

Comment:
Does it bring to window to the front even when it was not minimized, but covered by another window?